pkg/odo/cli/component/create.go

```diff
@@ -578,6 +578,8 @@ func (co *CreateOptions) Complete(name string, cmd *cobra.Command, args []string
 			// if it is not supported we still need to run all the codes related with s2i after devfile compatibility check
 
 			hasComponent := false
+			existSpinner := log.Spinner("Checking devfile existence")
```

Comment:
Can we rename this to `devfileExistSpinner`?

Reply:
sure

Reply:
done